src/main/java/randoop/reflection/ClassLiteralExtractor.java

```diff
@@ -17,23 +23,115 @@
  * @see OperationModel
  */
 class ClassLiteralExtractor extends DefaultClassVisitor {
-
+  /** Maps a type to a sequence. */
```

Comment:
What type?  What sequence?  Please give information beyond the variable type.  Also, it maps to a set of sequences, not to a single sequence.